src/components/check/ConfigureKnowledgeCheck.tsx

```
@@ -0,0 +1,34 @@
+    mode: 'edit'
+  }
+
+export function ConfigureKnowledgeCheck({ mode, initialStoreProps, options }: CreateProps | EditProps = { mode: 'create' }) {
+  return (
+    <CheckStoreProvider>
+      <PageHeading title={`${mode === 'create' ? 'Create KnowledgeCheck' : initialStoreProps.name}`} />
+      <div className='grid grid-cols-1 gap-8 lg:grid-cols-[repeat(auto-fill,minmax(680px,1fr))]'>
+        <GeneralSection />
+        <QuestionsSection />
+        <SettingsSection />
+        <Card className='h-60 break-inside-avoid' disableHoverStyles children={undefined} />
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)